RELEASE-NOTES.rst

```diff
@@ -11,25 +11,33 @@ Dictdiffer is a helper module that helps you to diff and patch
 dictionaries.
 
 New features
-------------
+~~~~~~~~~~~~
 
-- Adds tolerance parameter used when user wants to treat closed values
-  as equals
+-- Adds support for comparing NumPy arrays.  (#68)
+-- Adds support for comparing mutable mappings, sequences and sets from
+-  `collections.abs` module.  (#67)
 
 Improved features
------------------
+~~~~~~~~~~~~~~~~~
+
+-- Updates package structure, sorts imports and runs doctests.
+
+Bug fixes
+~~~~~~~~~
 
-- Adds support for comparing numerical values and NaN. (#54) (#55)
+-- Fixes order in which handled conflicts are unified so that the
+- Merger's unified_patches can be always applied.
+-- Removes unused `Conflict.handled` field.
```

Comment:
The `Removes unused...` part should actually not be advertised, I guess `kwalitee prepare release` does not do the parsing job here correctly... We usually fix it by introducing empty `AMENDS` commits, see for example https://github.com/inveniosoftware/flask-cli/pull/13#issuecomment-219976929